src/project-2.js

```diff
@@ -44,62 +81,102 @@ const isPrime = (num) => {
   // hint: a prime number is only evenly divisible by itself and 1
   // hint2: you can solve this using a for loop
   // note: 0 and 1 are NOT considered prime numbers
+
+  if (num === 0 || num === 1) {
+    return false;
+  }
+
+  for (let i = num - 1; i > 1; i--) {
+    if (num % i === 0) {
+      return false;
+    }
+  }
+
+  return true;
 };
 
 const returnFirst = (arr) => {
   // return the first item from the array
+  return arr.shift();
```

Comment:
Opportunity for refactor - mutation. `shift` mutates the parameter passed in and causes this method to have side effects. When working functionally, avoid all side-effects.

```javascript
const returnFirst = (arr) => arr[0];
```

Reply:
Great catch!